compat/client.js

```diff
@@ -1,13 +1,17 @@
-const { render, hydrate } = require('preact/compat');
+const { render, hydrate, unmountComponentAtNode } = require('preact/compat');
 
 exports.createRoot = function(container) {
```

Comment:
if you name this function `createRoot` you won't have to call `exports.createRoot` on line 16

Reply:
I got it. I will change the code.